### ISSUE TITLE
LicenseReportTask fails when run with gradle 7.2

### ISSUE THREAD

**Issue by strooooke** [jaredsburrows/gradle-license-plugin#173]
and/or android-gradle-plugin 7.1.2, but runs fine with gradle 7.0.2 and  AGP 7.0.3.

Stacktrace:

```
Caused by: org.xml.sax.SAXParseException; systemId: file:///[]REDACTED/.gradle/caches/modules-2/files-2.1/androidx.annotation/annotation/1.3.0/21f49f5f9b85fc49de712539f79123119740595/annotation-1.3.0.jar; lineNumber: 1; columnNumber: 1; Content is not allowed in prolog.
	at java.xml/com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1243)
	at java.xml/com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:635)
	at com.jaredsburrows.license.LicenseReportTask.generatePOMInfo(LicenseReportTask.kt:171)
	at com.jaredsburrows.license.LicenseReportTask.licenseReport(LicenseReportTask.kt:63)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:104)
	... 118 more

```

I'm pretty sure that `androidx.annotation:annotation` is just the first one in the list - and yes, that's the jar, not the pom, that won't parse as XML. But I have insufficient understanding of gradle to be of any further help, sadly.

**Comment by rgocal** [comment]
any update on this?

**Comment by kratho** [comment]
The issue seems to be fixed with version 0.8.91 and #172

**Comment by jaredsburrows** [comment]
This should be fine. The current build is built with Gradle `7.4.2` and is tested with it as well: https://github.com/jaredsburrows/gradle-license-plugin/blob/master/src/test/groovy/com/jaredsburrows/license/LicensePluginAndroidSpec.groovy#L90.
